be/src/util/timezone_utils.cpp

```diff
@@ -78,17 +84,55 @@ bool TimezoneUtils::find_cctz_time_zone(const std::string& timezone, cctz::time_
         offset *= positive ? 1 : -1;
         ctz = cctz::fixed_time_zone(cctz::seconds(offset));
         return true;
-    } else if (timezone_lower == "cst") {
-        // Supports offset and region timezone type, "CST" use here is compatibility purposes.
-        ctz = cctz::fixed_time_zone(cctz::seconds(8 * 60 * 60));
-        return true;
-    } else {
-        auto it = timezone_names_map_.find(timezone_lower);
-        if (it == timezone_names_map_.end()) {
-            return false;
+    } else { // not only offset, GMT or GMT+8
+        // split tz_name and offset
+        int split = timezone_lower.find('+') != std::string::npos ? timezone_lower.find('+')
+                                                                  : timezone_lower.find('-');
+        cctz::time_zone offset;
+        bool have_both = split != std::string::npos && split + 1 < timezone_lower.length() &&
+                         std::isdigit(timezone_lower[split + 1]);
+        if (have_both) {
+            auto offset_str = timezone_lower.substr(split);
+            timezone_lower = timezone_lower.substr(0, split);
+            int offset_hours = 0;
+            try {
+                offset_hours = std::stoi(offset_str);
+            } catch ([[maybe_unused]] std::exception& e) {
+                LOG(INFO) << "Unable to cast " << timezone << " as timezone";
```

Comment:
Use LOG(WARNING) if you think this is a curial message to let users know otherwise LOG(DEBUG). INFO doesn't make sense here